lib/github_changelog_generator.rb

```diff
@@ -53,7 +40,23 @@ def initialize
 
       fetch_event_for_issues_and_pr
       detect_actual_closed_dates
-      @tag_times_hash = {}
+    end
+
+    # Return tags after filtering tags in lists provided by option: --between-tags & --exclude-tags
+    #
+    # @return [Array]
+    def get_filtered_tags
+      all_tags = @fetcher.get_all_tags
+      filtered_tags = []
+      if @options[:between_tags]
+        @options[:between_tags].each do |tag|
+          unless all_tags.include? tag
+            puts "Warning: can't find tag #{tag}, specified with --between-tags option.".yellow
```

Comment:
Line is too long. [95/80]<br>Do not write to stdout. Use Rails' logger if you want to log.